library/common.hpp

```diff
@@ -58,6 +59,28 @@ inline static int safe_call(Class *obj, Method method, Args &&...args)
 	}
 }
 
+template <typename LoggerRef>
+inline static void c_exception_guard_log(const LoggerRef &logger_ref, const char *func_name,
```

Comment:
`const char *a` -> `const char a[]` and below

Reply:
Don't see such a style in the project. Is it google style?

Reply:
I'm not sure. Ok, leave it at that.